RealmSwift/Combine.swift

```diff
@@ -57,6 +57,18 @@ extension ObjectKeyIdentifiable {
     }
 }
 
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+extension Projection: Identifiable, Hashable {
+    /// A stable identifier for this projection. This value will be
+    /// the same for all projections of this object
+    public var id: UInt64 {
+        // UInt64(bitPattern:) to properly cast negative Int64 values
+        // It will use the same memory representation, the numeric value
+        // may be different which is fine in this case
+        UInt64(bitPattern: Int64(ObjectIdentifier(type(of: self)).hashValue))
```

Comment:
I don't think this is valid. `hashValue` is allowed to have collisions, but `id` is not. `return withUnsafePointer(to: self) { UInt(bitPattern: $0) }` looks to be the correct way to do this.